bookkeeper-server/src/main/java/org/apache/bookkeeper/client/DefaultBookieAddressResolver.java

```diff
@@ -59,13 +59,13 @@ public BookieSocketAddress resolve(BookieId bookieId) {
                 log.debug("Resolving dummy bookie Id {} using legacy bookie resolver", bookieId);
                 return BookieSocketAddress.resolveDummyBookieId(bookieId);
             }
-            log.info("Cannot resolve {}, bookie is unknown", bookieId, ex);
+            log.debug("Cannot resolve {}, bookie is unknown", bookieId, ex.toString());
```

Comment:
It is important to know of an address resolution error, I'd not make this `debug`, I'd say it is at least `info` if not `warn`.

Also, removing the stack traces certainly makes sense for the shell, but they might be useful for general debugging.

Reply:
that was my thought when I first wrote this code, but actually we are appending the exception to the chain of the thrown BookieIdNotResolvedException, so we are going to report this error twice (here and in the caller).
I can leave it at "info" level, but the stacktrace is redundant this is case